lxd/main_nsexec.go

```diff
@@ -99,6 +99,23 @@ int dosetns(int pid, char *nstype) {
 	return 0;
 }
 
+int dosetns_file(char *file, char *nstype) {
+	__do_close_prot_errno int mntns = -EBADF;
+
+	mntns = open(file, O_RDONLY);
+	if (mntns < 0) {
+		error("error: open mntns");
```

Comment:
Error message is specific to mount namespaces while function appears namespace generic.

Reply:
@brauner will apply same change to dosetns function above I based this on.